api/camunda/camunda-openapi.yaml

```diff
@@ -4508,6 +5142,73 @@ components:
         email:
           description: The email of the user.
           type: "string"
+    MappingFilterRequest:
+      description: Mapping search filter.
+      type: object
+      properties:
+        claimName:
+          type: string
+          description: The claim name to match against a token.
+        claimValue:
+          type: string
+          description: The value of the claim to match.
+    CamundaUser:
+      type: "object"
+      properties:
+        userId:
+          description: The ID of the user.
+          type: "string"
+        userKey:
+          description: The system generated key of the User
```

Comment:
The descriptions on this schema should either all use periods, or never use periods. Preference is all periods, but if it's consistent I usually let none periods slide. It's not consistent though, so I'll PR a change upstream for all periods.